### ISSUE TITLE
Delosperma cooperi has the forma included in the authorship

### ISSUE THREAD

**Issue by mdoering** [rdmpage/the-plant-list-1-1#1]
https://github.com/rdmpage/the-plant-list-1-1/blob/master/fetch/csv/D/Delosperma.csv#L43

Looks like this comes from the source already :(
http://www.theplantlist.org/tpl1.1/record/tro-701425

**Comment by rdmpage** [comment]
@mdoering Is this in relation to https://github.com/gbif/checklistbank/issues/2#issuecomment-358931395 ?

**Comment by mdoering** [comment]
Yes, I discovered it when looking up that author. Not required for the fix of the CLB issue though. Just thought I let you know
